tests/storages_tests/test_with_server.py

```diff
@@ -122,10 +119,15 @@ def test_store_infinite_values(input_value: float, expected: float, storage_url:
         storage = optuna.storages.RDBStorage(url=storage_url)
     study_id = storage.create_new_study()
     trial_id = storage.create_new_trial(study_id)
-    storage.set_trial_intermediate_value(trial_id, 1, input_value)
-    storage.set_trial_state_values(trial_id, state=TrialState.COMPLETE, values=(input_value,))
-    assert storage.get_trial(trial_id).value == expected
-    assert storage.get_trial(trial_id).intermediate_values[1] == expected
+    storage.set_trial_intermediate_value(trial_id, 1, value)
+
+    got_value = storage.get_trial(trial_id).intermediate_values[1]
+    assert got_value == value or (np.isnan(got_value) and np.isnan(value))
+
+    if not np.isnan(value):
+        # nan is not acceptable as the final value.
```

Comment:
This line should be moved above the line 127. In addition, the sentence should be started with an uppercase letter.
```suggestion
        # A `nan` is not acceptable as the final value.
```